dart/lib/src/client.dart

```diff
@@ -1,3 +1,4 @@
+import 'dart:async';
```

Comment:
@rxlabz why do we need this now?

Reply:
well... not really. I made it to respect the https://develop.sentry.dev/sdk/philosophy/ principles : in the pub spec, we allow dart ">=2.0.0 < 3.0.0" but if someone use an sdk < 2.1.0 he will need `import 'dart:async';`

Reply:
ahhh, I see, that makes sense, I'm fine with it then :) I thought it was a typo